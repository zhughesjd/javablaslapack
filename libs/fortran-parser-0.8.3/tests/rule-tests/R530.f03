
! Testing data stmt value list. Similar to R526.
! The following doesn't work, though it should.
data truth, beauty, ugly / 3.14159,2.81828,6 /
data foo / 3 /
data bar, this, that, theother / 2*3,2*7.6 /
end
