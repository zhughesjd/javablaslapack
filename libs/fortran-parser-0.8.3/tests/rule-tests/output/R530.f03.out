1101:begin
526:list-begin
613 truth false false
612 1
603 false
601
613 beauty false false
612 1
603 false
601
613 ugly false false
612 1
603 false
601
526:list 3
530:list-begin
417 3 .14159 null
416 null
417 2 .81828 null
416 null
406 6 null
530:list 3
526:list-begin
613 foo false false
612 1
603 false
601
526:list 1
530:list-begin
406 3 null
530:list 1
526:list-begin
613 bar false false
612 1
603 false
601
613 this false false
612 1
603 false
601
613 that false false
612 1
603 false
601
613 theother false false
612 1
603 false
601
526:list 4
530:list-begin
406 2 null
406 3 null
405 null
406 2 null
417 7 .6 null
416 null
530:list 2
1103
1101 false false false
